frontend/public/components/_terminal.scss

```diff
@@ -36,3 +36,7 @@ $console-collapse-link-z-index: $console-z-index + 20;
 .default-overflow {
   overflow: visible !important;
 }
+
+.padding-info-message {
```

Comment:
Should I rename this or put it somewhere else.  Or is there something else already in use to pad the alert message?